cmd/README.md

```diff
@@ -17,13 +17,21 @@ For additional details, please refer to the [commatrix documentation](https://gi
 
 ---
 
-## Running
-build the code locally and install it in `/usr/local/bin`:
+## Install
+This plugin can be installed with the [Krew](https://sigs.k8s.io/krew) plugin manager:
+
+```sh
+oc krew install --manifest=cmd/commatrix-krew.yaml
```

Comment:
As this README is in cmd dir can point directly to commatrix-krew.yaml
I'm still wondering if a user should be able to install the plugin without pointing to a manifest